__tests__/app.test.js

```diff
@@ -91,4 +92,32 @@ describe("GET /api/articles/:article_id", () => {
           expect(body.msg).toBe("Bad request");
         });
     });
-  })
\ No newline at end of file
+  })
+  describe("GET /api/articles", () => {
+    test("200: Responds with an array of article objects sorted by date in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+  
+          expect(articles).toHaveLength(articles.length); 
```

Comment:
This assertion is why the red green refactor cycle is very important to follow - did you see this assertion fail? Probably not as this expect statement can never fail? Consider why and what you might need to do fix the statement!